assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -1466,6 +1467,14 @@ private String build_student_view_submission_context(VelocityPortlet portlet, Co
                 		state.setAttribute("plagiarismFileTypes", rb.getFormattedMessage("gen.onlythefoll", getContentReviewAcceptedFileTypesMessage())); 
                     context.put("plagiarismFileTypes", state.getAttribute("plagiarismFileTypes"));
 
+                    String reviewServiceName = contentReviewService.getServiceName();
+                    if ("Urkund".equals(reviewServiceName)) {
```

Comment:
I'm not sure why this is specific to just the Urkund service. I'm also not a fan of hardcoding this string.

Reply:
As far as I know, Urkund is the only one that allows to modify the maximum file size. So it needs to be "filtered" by the content review provider. In other parts of the code also makes this making a comparison by a hardcoded string. I'm going to change this code fragment a bit.